package.json

```diff
@@ -120,6 +120,7 @@
     "identity-obj-proxy": "^3.0.0",
     "postprocessing": "^6.32.2",
     "prettier": "^3.0.0",
+    "r3f-perf": "^7.1.2",
```

Comment:
Saw that there's another performance measuring plugin available

Reply:
It's rendering both and both gives the same info (yours much more) maybe let's remove the old one?

![image](https://github.com/readyplayerme/visage/assets/18527021/ca39cc85-7430-4425-b96c-4da1c6db4920)